improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -44,7 +44,7 @@
 
 from improver import BasePlugin
 from improver.ensemble_calibration.utilities import (
-    check_predictor_of_mean_flag, convert_cube_data_to_2d,
+    check_predictor, convert_cube_data_to_2d,
```

Comment:
Looking at the sphinx documentation, another way of organising this documentation could be putting all the common EMOS equation in a module docstring. Any thoughts?

Reply:
I think you have lots of really useful information in the extended documention. It might be worth looking at the sphinx documentation for this module as a whole and see if you think it is all in the right place.

Reply:
I've tried to centralise the documentation into a module level docstring.